### PR TITLE
Second batch replacing cfaforecastrenewalww

### PR DESCRIPTION
**Pull request by gvegayon**
- Moves cfa.../R/forecasttools.R to wweval/R
- Moves cfa.../data-raw/flusight_location_table to wweval.
- Adds missing entries in roxygen2 comments to avoid warnings in R CMD check for wweval.
- **Sets the `remotes` R package of `cfaforecastrenewalww` to point to this branch**
- Moves delay_distributions.R to wweval.

### CODE REVIEW

**Comment by gvegayon** [comment]
Hey @kaitejohnson, this is ready for a run on Azure and your review.

File: wweval/R/delay_distribs.R [hunk 1]
```diff
@@ -1,13 +1,12 @@
 #' Get the generation interval from a gamma distribution
 #'
-#' @param mean_gi
-#' @param stdev_gi
-#' @param gt_max
+#' @param mean_gi NEEDS DOCUMENTATION.
```
Comment (kaitejohnson) [inline wweval/R/delay_distribs.R]: I think ultimately we wouldn't use these

File: wweval/R/delay_distribs.R [hunk 2]
```diff
@@ -215,13 +214,12 @@ make_incubation_period_pmf <- function(backward_scale = 3.60,
 #' admission. See below:
 #' https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0261428
 #'
-#' @param neg_binom_mu
-#' @param neg_binom_size
+#' @param neg_binom_mu NEEDS DOCUMENTATION.
```
Comment (kaitejohnson) [inline wweval/R/delay_distribs.R]: These are documented in `wwinference`

**Review by kaitejohnson — CHANGES_REQUESTED** [review]
I am super lost what you changed here but when I try to build the container I get this error:

```
Error: stat /home/uox1/P/gh_repos/wastewater-informed-covid-forecasting/Dockerfile: no such file or directory
``` 

Something that used to point to the Containerfile got deleted or removed? But I can't see that you changed anything in the `setup_container.R`. I checked and built the container in `prod` and it did work...

**Comment by gvegayon** [comment]
> Something that used to point to the Containerfile got deleted or removed? But I can't see that you changed anything in the setup_container.R. I checked and built the container in prod and it did work... 

I haven't changed that :S. I could try merging prod here and see if that fixes it?

File: wweval/R/delay_distribs.R (hunk 1 above)
Comment (gvegayon) [reply wweval/R/delay_distribs.R]: Which functions? All in `R/delay_distribs.R`?

File: wweval/R/delay_distribs.R (hunk 2 above)
Comment (gvegayon) [reply wweval/R/delay_distribs.R]: Same question here.

File: wweval/R/delay_distribs.R (hunk 1 above)
Comment (kaitejohnson) [reply wweval/R/delay_distribs.R]: The only ones that are actually being used are in here! There are lots in `cfaforecastrenewalww` that are not needed (also so many that are not documented) https://github.com/CDCgov/ww-inference-model/blob/65c358830b551c01cabe30c9a1a66a122128ea04/R/delay_distribs.R#L86

File: wweval/R/delay_distribs.R (hunk 2 above)
Comment (kaitejohnson) [reply wweval/R/delay_distribs.R]: https://github.com/CDCgov/ww-inference-model/blob/65c358830b551c01cabe30c9a1a66a122128ea04/R/delay_distribs.R#L132

**Comment by kaitejohnson** [comment]
> > Something that used to point to the Containerfile got deleted or removed? But I can't see that you changed anything in the setup_container.R. I checked and built the container in prod and it did work...
> 
> I haven't changed that :S. I could try merging prod here and see if that fixes it?

Ah yes we just need to merge prod! Because we added the Containerfile to git, its git tracked so is no longer showing up when I switch branches https://github.com/CDCgov/wastewater-informed-covid-forecasting/blob/prod/Containerfile

**Comment by kaitejohnson** [comment]
Closing with #182

**Pull request closed without merging**